jarbas/core/management/commands/companies.py

```diff
@@ -45,6 +48,12 @@ def save_companies(self):
                 self.count += 1
                 self.print_count(Company, count=self.count)
 
+    @staticmethod
+    def transform(row):
+        return row._asdict()
+
+
+
```

Comment:
We don't need these extra spaces: according to PEP8 1 line is fine to separate class methods.